frontend/packages/ux-editor/src/components/Properties/PropertiesHeader/EditLayoutSetForSubform/EditLayoutSet/EditLayoutSet.tsx

```diff
@@ -2,19 +2,30 @@ import React, { useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { DefinedLayoutSet } from './DefinedLayoutSet/DefinedLayoutSet';
 import { SelectLayoutSet } from './SelectLayoutSet/SelectLayoutSet';
-import { StudioRecommendedNextAction } from '@studio/components';
+import { StudioProperty, StudioRecommendedNextAction } from '@studio/components';
+import { Paragraph } from '@digdir/designsystemet-react';
+import { PlusIcon } from '@studio/icons';
+import classes from './EditLayoutSet.module.css';
+import { CreateNewLayoutSet } from './CreateNewLayoutSet/CreateNewLayoutSet';
 
 type EditLayoutSetProps = {
   existingLayoutSetForSubform: string;
   onUpdateLayoutSet: (layoutSetId: string) => void;
+  onSubFormCreated: (layoutSetName: string) => void;
 };
 
 export const EditLayoutSet = ({
   existingLayoutSetForSubform,
   onUpdateLayoutSet,
+  onSubFormCreated,
 }: EditLayoutSetProps): React.ReactElement => {
   const { t } = useTranslation();
   const [isLayoutSetSelectorVisible, setIsLayoutSetSelectorVisible] = useState<boolean>(false);
+  const [showCreateSubform, setShowCreateSubform] = useState<boolean>(false);
+
+  function handleClick() {
+    setShowCreateSubform(!showCreateSubform);
```

Comment:
Using the NOT operator here might make sense, but is there any case where this should not set `showCreateSubform` to `true`? If not, I think it's safer to just set it to `true` to avoid any potential issues (for example if somehow a queue of two clicks happens and `showCreateSubform` is set back to `false`)

Reply:
Thats right! 😊 I fixed